src/consent-manager-builder/analytics.ts

```diff
@@ -59,6 +67,26 @@ export default function conditionallyLoadAnalytics({
 
   // Don't load a.js at all if nothing has been enabled
   if (isAnythingEnabled) {
+    // @ts-ignore
     wd.analytics.load(writeKey, { integrations })
+    wd.analytics.track = wrapTrack(wd.analytics.track, destinationPreferences, categoryPreferences)
+    // Only temporary for testing
+    wd.analytics.track('hiiii, goodbye')
```

Comment:
This track correctly has the consent manager properties in it: 
<img width="349" alt="Screen Shot 2020-04-20 at 9 38 48 AM" src="https://user-images.githubusercontent.com/31225471/79776892-45e85180-82eb-11ea-9534-579e4804117a.png">

Reply:
A track() call directly in the console is missing those properties: 
<img width="1101" alt="Screen Shot 2020-04-20 at 9 46 14 AM" src="https://user-images.githubusercontent.com/31225471/79777278-dc1c7780-82eb-11ea-8c01-4e45b3236c86.png">

<img width="340" alt="Screen Shot 2020-04-20 at 9 46 03 AM" src="https://user-images.githubusercontent.com/31225471/79777311-ea6a9380-82eb-11ea-8dc9-950356c532b3.png">